plugins/crossplane.yaml

```diff
@@ -1,21 +1,18 @@
 plugins:
```

Comment:
It would be great here to have a doc header to help user with installing the cli.